pkg/clusteragent/orchestrator/controller.go

```diff
@@ -125,6 +137,7 @@ func (o *Controller) processPods() {
 	extraHeaders := map[string]string{
 		api.HostHeader:           o.hostName,
 		api.ContainerCountHeader: "0",
+		api.ClusterIDHeader:      o.clusterID,
```

Comment:
for the regular process check you also need to update https://github.com/DataDog/datadog-agent/blob/80cc38bc22d67893971e438644063cc9f95c88a7/cmd/process-agent/collector.go#L138

Reply:
done already: https://github.com/DataDog/datadog-agent/blob/80cc38bc22d67893971e438644063cc9f95c88a7/cmd/process-agent/collector.go#L143-L145 from a previous PR: https://github.com/DataDog/datadog-agent/pull/5030/files#diff-f6d0ed82ccd165f9ca4bc2137c103938R144